src/dmd/frontend.h

```diff
@@ -6496,6 +6559,7 @@ struct Target
     bool isReturnOnStack(TypeFunction* tf, bool needsThis);
     uint64_t parameterSize(const Loc& loc, Type* t);
     void applyInRefParams(TypeFunction* tf);
+private:
     enum class TargetInfoKeys
```

Comment:
Rather than marking it private, it shouldn't be emitted at all.

Reply:
Probably better to emit it just in case. We could detect if they are used outside for public declarations (but im not sure if that is worth it....)